### ISSUE TITLE
Active stepped-list state

### ISSUE THREAD

**Issue by anthonydillon** [canonical/vanilla-framework#3758]
## Pattern to amend
We have cases where highlighting an active step would be useful.

## Visual
![image](https://user-images.githubusercontent.com/1413534/119334095-9b2b2b00-bc82-11eb-9b79-b638fe906c83.png)

## Context
Tutorials: https://ubuntu.com/tutorials/how-to-ubuntu-core-recovery-mode#1-overview

**Comment by bartaz** [comment]
Not exactly same pattern, but snapcraft has similar use case and they use accordion for that:

![Screenshot 2021-05-24 at 14.51.48.png](https://images.zenhubusercontent.com/593a660851aa2c30578102bb/13c15633-631d-48f1-885a-c780773e3f5f)

**Comment by lyubomir-popov** [comment]
@anthonydillon would it work for you if we apply the default active opacity on a class called is-active?

![image.png](https://images.zenhubusercontent.com/59d2496bb0222d5de479ea17/a065bde9-2607-4039-a76e-ea2e269eed67)

**Comment by anthonydillon** [comment]
Yeah sure, what way to show the currently active item. Not sure it's very clear from your image. But something like that.

**Comment by lyubomir-popov** [comment]
Ok, we ca ntake the colours freom the chips, which have the same base colour. The active will be a little darker. This is ready for dev then @bartaz.

**Comment by clagom** [comment]
@amylily1011 is going to work on this as a pattern for this cycle

**Comment by clagom** [comment]
WG: work planned for the 28th of March

**Comment by clagom** [comment]
@amylily1011 can you please update here with the link to the docs, then close this issue, thanks

**Comment by amylily1011** [comment]
This is the spec for the stepper: https://docs.google.com/document/d/1jh0eUSRtjEe-qqQTXv1rtZGvnOqOT29o53EVDd8g4dw/edit

**Comment by clagom** [comment]
WG: @lidia-luna is going to organise usability testing to move this forward.

**Comment by danielmutis** [comment]
Reopening this issue as the testing was done and got a bit forgotten but the component is still very much needed. The results of the testing are here. They show that, mostly, the stepper works and is designed as expected. The only aspect that is not clear is whether the past steps are clickable and whether they'd lose progress by going back.

**Comment by bartaz** [comment]
@danielmutis What's the latest status of this? Is this ready for implementation?

**Comment by danielmutis** [comment]
Yes, it's ready to implement. If anyone wants to amend the component they can submit issues independently.

**Comment by bartaz** [comment]
Triaging: effort low/medium, impact low/medium. 

Do we need this now, or can it wait? Needs decision.

**Comment by dgtlntv** [comment]
Design systems contribution meeting:
- We consider this to be ready for development.
